pkg/operator/admission/server.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package admission
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/pkg/clusterd"
+	cephadmission "github.com/rook/rook/pkg/operator/ceph/admission"
+	"net/http"
+	"path/filepath"
+)
+
+const (
+	tlsDir      = "/etc/webhook"
```

Comment:
This is already defined as a const (`secretMountPath`) with another name in a different file. Why here also? TBH, I would think `tlsDir` is the better named const, and we instruct the secret to use that dir.